tests/Adapter/SymfonyCacheTest.php

```diff
@@ -117,7 +117,7 @@ public function testCacheAction(): void
         $this->assertSame(200, $response->getStatusCode(), 'Response should be 200');
         $this->assertSame('ok', $response->getContent(), 'Response should return "OK"');
 
-        $this->assertSame('2', $response->headers->get('Content-Length'));
+        $this->assertEquals(2, $response->headers->get('Content-Length'));
```

Comment:
flintCI has an issue with this, why did you change it to `assertEquals`?

Reply:
Using `assertSame` with a string fails when using `--prefer-lowest` in composer (installing Symfony 4.3), and using an int fails with Symfony 5. So this was the easiest way to get the test to pass in both versions of Symfony

Reply:
Seems like it might have been a bug in Symfony 4.3. From 4.3.4 onwards, this works fine with a string, but 4.3.0 - 4.3.3 works only with an int.
I have updated composer.json to set the minimum required version for http-foundation to 4.3.4